bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/graphics/Device.java

```diff
@@ -33,6 +33,7 @@
  * @see <a href="http://www.eclipse.org/swt/">Sample code and further information</a>
  */
 public abstract class Device implements Drawable {
+	static boolean strictChecks = System.getProperty("org.eclipse.swt.internal.enableStrictChecks") != null;
```

Comment:
Gtk already has this in the `Display` class, so we should either use or move the property from there.

Reply:
If I see correctly you moved the constant to the Device class and made it protected (not packaged protected), which effectively makes them part of the API. This is why it would require a `@since` tag. And the build currently also fails because of visibility issues. Since the constant obviously needs to be accessed from multiple packages and since it is not OS-specific anymore, it might make sense to move it to some OS-common class of `org.eclipse.swt.internal` (maybe just a new one if none of the existing fits).